docs/UserGuide.md

```diff
@@ -63,10 +63,19 @@ Examples:
 Lists all currently supports commands, and provides a link to this user guide.<br>
 Format: `help`
 
-### Listing all anime: `list`
+### Listing anime: `list`
 
 Shows a list of all anime names the user has added.<br>
-Format: `list`
+An additional status parameter can be specified to show only anime with that status.<br>
```

Comment:
I think this would be clearer as it suggests to the user that the list command can switch between tabs, rather than listing animes w different statuses on the same tab. Maybe we should also make clear what the tabs are for in the UG later on.
```suggestion
An additional status parameter can be specified to switch tabs, showing only anime with that status.<br>
```